src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2350,6 +2350,13 @@ private CorInfoHelpFunc getUnBoxHelper(CORINFO_CLASS_STRUCT_* cls)
             return (void*)IntPtr.Zero;
         }
 
+#pragma warning disable CA1822 // Mark members as static
+        private CORINFO_CLASS_STRUCT_* getObjectType(void* typeObj)
```

Comment:
This should be implemented by fetching the type from `FrozenStringNode` / `FrozenObjectNode`.